cmd/wallet-sdk-js/jsinterop/types/openid4ci.go

```diff
@@ -0,0 +1,68 @@
+//go:build js && wasm
+
+/*
+Copyright Gen Digital Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package types
+
+import (
+	"fmt"
+	"syscall/js"
+
+	diddoc "github.com/hyperledger/aries-framework-go/component/models/did"
+
+	"github.com/trustbloc/wallet-sdk/cmd/wallet-sdk-js/jsinterop/jssupport"
+	"github.com/trustbloc/wallet-sdk/cmd/wallet-sdk-js/walletsdk"
+	"github.com/trustbloc/wallet-sdk/pkg/models"
+)
+
+const (
+	openid4ciRequestCredentialWithPreAuth = "requestCredentialWithPreAuth"
+)
+
+func SerializeOpenID4CI(agentMethodsRunner *jssupport.AsyncRunner,
```

Comment:
Is this more like `SerializeOpenID4CIInteraction`?

Reply:
Or is it not exactly specific to an `Interaction`?

Reply:
yes it more like SerializeOpenID4CIInteraction, will rename